src/Microsoft.Health.Dicom.Functions/Management/PurgeHistoryOptions.cs

```diff
@@ -9,9 +9,9 @@
 
 namespace Microsoft.Health.Dicom.Functions.Management
 {
-    public class OrchestrationHistoryConfiguration
+    public class PurgeHistoryOptions
```

Comment:
use `PurgeHistoryConfiguration` for consistency?

Reply:
In the future, I'd like to use the word "Options" to refer to these sorts of classes to divorce them conceptually from the configuration file